test/e2e/replica_set_tls/replica_set_tls_test.go

```diff
@@ -30,7 +30,7 @@ func TestReplicaSetTLS(t *testing.T) {
 	}
 
 	if err := setup.CreateTLSResources(mdb.Namespace, ctx); err != nil {
-		t.Fatalf("Failed to set up TLS resources: %+v", err)
+		t.Fatalf("Failed to set up TLS resources: %s", err)
```

Comment:
Seeing this and referring to my other two comments, maybe using `t.Fatalf` every time would be better? I feel we have a mixture of three different approaches at the moment:

`t.Fatalf`
`t.Fatal(fmt.Sprintf(...))`
`t.Fatal(errors.Errorf(...))`

Reply:
yes I agree! I will update to use one. I think `t.Fatalf` is the most straightfoward, what do you think?

Reply:
I think so! No need to nest function calls!